Brokerages/ApiPriceProvider.cs

```diff
@@ -54,19 +54,8 @@ public ApiPriceProvider(int userId, string userToken)
         /// <returns>The latest price</returns>
         public decimal GetLastPrice(Symbol symbol)
         {
-            var result = _api.ReadPrices(new[] { symbol });
-            if (!result.Success)
-            {
-                throw new Exception($"ReadPrices error: {string.Join(" - ", result.Errors)}");
-            }
-
-            var priceData = result.Prices.FirstOrDefault(x => x.Symbol == symbol);
-            if (priceData == null)
-            {
-                throw new Exception($"No price data available for symbol: {symbol.Value}");
-            }
-
-            return priceData.Price;
+            //NOP ReadPrices endpoint has been removed
+            throw new Exception("Prices endpoint is no longer supported");
```

Comment:
nit - might be more precise to throw something like `InvalidOperationException`

Reply:
Gotcha!